src/config/selectors.ts

```diff
@@ -15,3 +15,8 @@ export const USER_MENU_SIGN_OUT_OPTION_ID = 'userMenuSignOutOption';
 
 export const DELETE_MEMBER_DIALOG_TITLE_ID = 'alert-dialog-title';
 export const DELETE_MEMBER_DIALOG_DESCRIPTION_ID = 'alert-dialog-description';
+export const BUTTON_SHOW_EDIT_FORM_ID = 'button';
+export const BUTTON_SAVE_USERNAME_ID = 'save-username';
+export const BUTTON_CANCEL_EDITING_ID = 'cancel-editing';
+export const CURRENT_USERNAME_ID = 'current-username';
+export const MEMBER_NAME_TEXT_ID = 'member-name';
```

Comment:
I made a suggestion for these, actually I want you to apply the naming suggested. `BUTTON_SHOW_EDIT_FORM_ID` is too broad. Use more specific names. Also they should be consistent with one another. 

Here you have `CURRENT_USERNAME_ID` and `MEMBER_NAME_TEXT_ID`, they both identify the username UI element, one when it is editable, and  the other when it is simply displaying. They should have something In common in the name don't you think ?

Also you actual ids are too generic. `button` is not a good id to use. 

Below are values I think follow these guidelines. Do you agree that they:
- are easier to find and reference
- express better the intent
- are consistent
```suggestion
export const USERNAME_EDIT_BUTTON_ID = 'username-edit-button';
export const USERNAME_SAVE_BUTTON_ID = 'username-save-button';
export const USERNAME_CANCEL_BUTTON_ID = 'username-cancel-button';
export const USERNAME_INPUT_FIELD_ID = 'username-input-field';
export const USERNAME_DISPLAY_ID = 'username-display';
```
Please use these names.